atomix/cluster/src/main/java/io/atomix/raft/roles/PassiveRole.java

```diff
@@ -316,6 +320,58 @@ public CompletableFuture<ReconfigureResponse> onReconfigure(final ReconfigureReq
     }
   }
 
+  @Override
+  public CompletableFuture<ForceConfigureResponse> onForceConfigure(
+      final ForceConfigureRequest request) {
+    logRequest(request);
+    updateTermAndLeader(request.term(), null);
+
+    final var currentConfiguration = raft.getCluster().getConfiguration();
+
+    // No need to overwrite if already in force configuration. This can happen due to retry.
+    if (currentConfiguration == null
```

Comment:
:thought_balloon: Just for my understanding, this reads as: if there is no current config (likely we just started?), or we're not already forcing, _or_ we are already forcing but the request index is greater - that last one means a later force configuration request was sent? Or we somehow never got out of force configuring :thinking: What are the other cases where the request index would be higher?